ydb/core/mind/local.cpp

```diff
@@ -111,6 +111,9 @@ class TLocalNodeRegistrar : public TActorBootstrapped<TLocalNodeRegistrar> {
     ui64 MemLimit = 0;
     double NodeUsage = 0;
 
+    TInstant LastUpdate;
```

Comment:
The name is confusing, last update of what? Also If we change it to use TabletMetrics message, there already is a SendTabletMetricsTime member